classes/invitation/invitations/ReviewerAccessInvite.php

```diff
@@ -119,9 +118,8 @@ private function _validateAccessKey(): bool
         $reviewId = $reviewAssignment->getId();
 
         // Check if the user is already logged in
-        $sessionManager = SessionManager::getManager();
-        $session = $sessionManager->getUserSession();
-        if ($session->getUserId() && $session->getUserId() != $this->userId) {
+        $session = Application::get()->getRequest()->getSession();
+        if ($session->has('user_id') && $session->get('user_id') != $this->userId) {
```

Comment:
Elsewhere in the system (even in Eloquent models) we've been careful to maintain `snake_case` in the database and `lowerCamelCase` in the PHP implementation. Laravel seems less concerned about this, but at least for now, I'd prefer to keep that convention. I'm not sure whether the session model is expected by other parts of Laravel to have e.g. `user_id` rather than `userId`, but if it's possible to easily maintain the use of `userId` rather than `user_id` outside of the model code, could you make that change?